src/components/Article/article.test.js

```diff
@@ -7,8 +7,5 @@ it('it should render single article view', () => {
   expect(wrapper.length).toBe(1);
   expect(wrapper.find('container')).toBeDefined();
 });
-it('it should match snapshot', () => {
```

Comment:
Why did you remove this test?

Reply:
Hi @jomasim,
This may have been accidental but not for any particular reason.
My apology.

Regards
Steven Ennis